clouddriver-cloudrun/src/main/java/com/netflix/spinnaker/clouddriver/cloudrun/deploy/ops/DeployCloudrunAtomicOperation.java

```diff
@@ -110,6 +120,26 @@ public String deploy(String repositoryPath) {
     return versionName;
   }
 
+  private void populateCloudrunYmlData(List<String> configFiles) throws JsonProcessingException {
+
+    ObjectMapper yamlReader = new ObjectMapper(new YAMLFactory());
+    for (String configFile : configFiles) {
+      CloudrunService yamlObj = yamlReader.readValue(configFile, CloudrunService.class);
+      if (yamlObj != null && yamlObj.getMetadata() != null && yamlObj.getSpec() != null) {
+        ymlData.setKind(yamlObj.getKind());
```

Comment:
Shouldn't this method return the updated YmlData object rather than using a class variable?

Reply:
YmlData object  will be populated with the updated data only.

Reply:
@mattgogerly Please can you review the changes?

Reply:
@mattgogerly Please can you review the changes?